src/ontology/mondo-ingest.Makefile

```diff
@@ -91,7 +91,7 @@ $(COMPONENTSDIR)/ordo.owl: $(TMPDIR)/ordo_relevant_signature.txt config/properti
 			--update ../sparql/ordo-construct-subsets.ru \
 			--update ../sparql/ordo-construct-d2g.ru \
 			--update ../sparql/ordo-replace-annotation-based-mappings.ru \
-		filter -T $(TMPDIR)/ordo_relevant_signature.txt --trim false \
+		remove -T $(TMPDIR)/ordo_relevant_signature.txt --select complement --select "classes individuals" --trim false \
```

Comment:
I'm guessing there are a few things in this PR that are not specifically related to the ODK version up? Or maybe this one is related to a `robot` version up.

Reply:
This line caused an issue where some important pieces of the module where removed (synonym types), which caused the new version of oak to fail. This is the right way of doing it anyways, other ingests use the same filter step.